ProQuest.js

```diff
@@ -499,6 +517,61 @@ function getItemType(types) {
 	return guessType;
 }
 
+function scrapeEbraryResults(refs) {
+	// Since we can't chase URLs, let's get what we can from the page
+	
+	for(var i = 0; i < refs.length; i++) {
+		var ref = refs[i];
+		var hiddenData = ZU.xpathText(ref.html, './span');
+		var visibleData = ZU.xpathText(ref.html.getElementsByClassName('results_list_copy'), './text()').trim().split(';').reverse();
```

Comment:
Yes, there's a variable number of authors. This particular text string looks like this:
author, first; author, second; author, third; publisher name; publisher location (date)

Unfortunately, this string is not usefully described or broken up by span elements in markup. It was more efficient to split the string on ; and parse it in reverse, since there's no way to predict how many authors a book will have. I can include a comment informing the structure of the string we're looking at on the page.

The data in this string also exists within the same element that hiddenData comes from, so we could extract it that way instead. However, my dev team talked about it during our last scrum, and apparently the PQ platform is still updating their UI. There was some worry that the invisible span with the metadata may not stick around.